src/lib/zkapp.ts

```diff
@@ -655,12 +687,12 @@ class SmartContract extends SmartContractBase {
    * ```ts
    * let tx = await Mina.transaction(sender, async () => {
    *   AccountUpdate.fundNewAccount(sender);
-   *   zkapp.deploy();
+   *   await zkapp.deploy();
    * });
    * tx.sign([senderKey, zkAppKey]);
    * ```
    */
-  deploy({
+  async deploy({
```

Comment:
`deploy()` needs to be async because it calls `init()`, which can be a method so might be async